crawl-ref/source/annotate.h

```diff
@@ -0,0 +1,17 @@
+/**
+ * @file
+ * @brief Annotation functionality
+**/
+
+#pragma once
+
+#include <vector>
+
+void annotate_level();
+void do_annotate(level_id li);
+//static int _prompt_annotate_branch(); // don't need need this?
```

Comment:
You don't (usually) want to declare static functions in header files, because they are only usable from the `cc` file in which they are defined. This is usually intentional; if you want code from other files to be able to call a function, it shouldn't be static.